src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,5 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+export const UserInfo = ({ user }) => (
```

Comment:
Since the user prop is used in this component, it should be validated using PropTypes to ensure the object structure is as expected.